skydance/protocol.py

```diff
@@ -408,11 +434,23 @@ class GetNumberOfZonesResponse(Response):
     # - Zone name
     # - Zone status (on/off)
 
+    def __init__(self, raw: bytes):
+        super().__init__(raw)
+        self._zones = []
+
+        for lbyte in self.cmdData:
+            if (lbyte & DEVICE_BASE_TYPE_NORMAL) == DEVICE_BASE_TYPE_NORMAL:
+                zoneId = lbyte & 0x1f
```

Comment:
What `0x1f` means? Can we perhaps add a documented constant for it?

Reply:
this is from decompiled source, in PR comment i mention this:

```
               for (byte b3 : skyDanceCmd.cmdData) {
                    if ((b3 & 0xff & DeviceBaseType.DEVICE_BASE_TYPE_NORMAL) == 0x80 && (b = b3 & 0x1f) > 0) {
                        SkyDanceDeviceInfo skyDanceDeviceInfo = new SkyDanceDeviceInfo();
```

seems 0x1f is just bitmask:b00011111 (in binary), and this means that max zones count may be 32 (from 0 to 31 in decimal)